<launch>
  <arg name="cmd_vel" default="/cmd_vel"/>
  <arg name="name" default="turtlebot3"/>

  <param name="cmd_vel_topic_name" value="$(arg cmd_vel)"/>

  <node name="$(arg name)" pkg="turtlebot3_gazebo" type="cluster_2d_pcl" required="true" output="screen"/>
</launch>
